generated/nidaqmx/nidaqmx.proto

```diff
@@ -5458,7 +5491,7 @@ message CreateAIRosetteStrainGageChanRequest {
     int32 rosette_type_raw = 7;
   }
   double gage_orientation = 8;
-  repeated int32 rosette_meas_types = 9;
+  repeated StrainGageRosetteMeasurementType rosette_meas_types = 9;
```

Comment:
Same as https://github.com/ni/grpc-device/pull/1115/files#r1831562654 except rosette strain gage is supported on more device models than multi-edge triggers.

Reply:
Based on Max's latest comment here: https://github.com/ni/grpc-device/pull/1115/files#r1831616740, I think he pointed out with a tag to this PR that source-breaking is acceptable, and we do not bump major version for source-breaking. Will drop this by marking it as resolved too, do notify me if otherwise.